index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>YouTube Player Page</title>
```

Comment:
It will be very ok if you put the charset, language, and viewport to your code in the head tag.